query-primitives/src/main/scala/io/shiftleft/queryprimitives/utils/MemberAccess.scala

```diff
@@ -0,0 +1,20 @@
+package io.shiftleft.queryprimitives.utils
+
+import io.shiftleft.codepropertygraph.generated.Operators
+
+object MemberAccess {
+
+  /**
+    * For a given name, determine whether it is the
+    * name of a "member access" operation, e.g.,
+    * "<operator>.memberAccess".
+    * */
+  def isGenericMemberAccessName(name: String): Boolean = {
```

Comment:
```suggestion
  private[queryprimitives] def isGenericMemberAccessName(name: String): Boolean = {
```
if you don't want to expose it outside of `queryprimitives`

Reply:
Currently, we can't do that yet, but when we can, I will, thx.